benchmark/benchmain/main.go

```diff
@@ -275,6 +275,16 @@ func makeClient(bf stats.Features) (testpb.BenchmarkServiceClient, func()) {
 		)
 	}
 	if bf.EnableKeepalive {
+		sopts = append(sopts,
+			grpc.KeepaliveParams(keepalive.ServerParameters{
+				Time:    keepaliveTime,
+				Timeout: keepaliveTimeout,
+			}),
+			grpc.KeepaliveEnforcementPolicy(keepalive.EnforcementPolicy{
+				MinTime:             keepaliveTime,
```

Comment:
Better do `keepaliveTime*0.8` or something so that we don't get a connection closed because of a minor timing issue.

Reply:
The server will wait for three pingStrikes before sending a GOAWAY for too_many_pings.
https://github.com/grpc/proposal/blob/master/A8-client-side-keepalive.md#server-enforcement

Reply:
I think I misunderstood what I read on the above proposal. Incorporated your comment now. Thanks.

Reply:
It's true that the "three strikes" policy would help to protect the connection in the event of minor timing issues.  However, there are also other pings in the system, e.g. for BDP calculation purposes.  On the other other hand, in these tests we expect the server to be (nearly) continuously sending data, resetting the ping strikes, so everything is probably fine if they are identical.  Still, it seems like a good idea to configure the server to be more tolerant than what the client is configured to do.

Reply:
Ack.